bitcoin/src/blockdata/script/borrowed.rs

```diff
@@ -481,14 +481,16 @@ crate::internal_macros::define_extension_trait! {
         }
```

Comment:
Why does the commit message mention fuzzing when I don't see it anywhere?

Reply:
oh the commit message is stale, I had this PR done differently in the past, will re-write. Thanks